core/src/main/scala/com/github/johnynek/paiges/Doc.scala

```diff
@@ -823,4 +838,19 @@ object Doc {
    */
   def stack(ds: Iterable[Doc]): Doc =
     intercalate(line, ds)
+
+  /**
+   * build a table with the strings left aligned and
+   * the Docs starting in the column after the longest string.
+   * The Docs on the right are rendered aligned after the rightSep
+   */
+  def tabulate(leftSep: String, fill: Char, rightSep: String, kv: List[(String, Doc)]): Doc =
```

Comment:
This is great! I think it does indeed solve the usecase for fillbreak that most people have.

A couple thoughts:

1) What's the usecase for leftSep being different than the fill?
2) We should document what leftSep and rightSep are used for.
3) Maybe we should have an overload which only takes the kv?